components/logicbutton/stories/logicbutton.stories.js

```diff
@@ -35,22 +38,43 @@ export default {
 		isDisabled: false,
 	},
 	parameters: {
-		componentVersion: version,
+		componentVersion: version,	
+		docs: {
+			story: {
+				height: "auto",
+			},
+		},
 	},
 };
 
 export const Default = Template.bind({});
 Default.args = {};
+Default.tags = ["!autodocs"];
+
+/**
+ * The default logic button is the and variant. 
```

Comment:
```suggestion
 * The default logic button is the "And" variant. 
```

Or some variation of that to make it a little easier to read, maybe? I'm a little fuzzy on the guidelines for quotes, but I think capitalization is allowed/encouraged here.

Reply:
Whoops, yes definitely: https://github.com/adobe/spectrum-css/pull/2884/commits/f4a78714197599fb82eea29ebbd09e173b38f66e

I didn't use the quotes because it sounds like [quotes are discouraged](https://spectrum.adobe.com/page/grammar-and-mechanics/#Quotation-mark) for interface elements? The quotes section says "only use quotation marks (“”) when quoting someone’s words or when referring to a file or asset name."